corehq/apps/reports/standard/deployments.py

```diff
@@ -307,6 +309,7 @@ def process_rows(self, users, fmt_for_export=False):
         for user in users:
             last_build = last_seen = last_sub = last_sync = last_sync_date = app_name = commcare_version = None
             last_build_profile_name = device = device_app_meta = num_unsent_forms = None
+            is_commcare_user = isinstance(user, CommCareUser)
```

Comment:
There's also a `is_commcare_user` function.

Reply:
Oh I didn't know.. so I guess I can do `user.is_commcare_user()` then as in some of the example I found in quick search

Reply:
Actually the `user` here is dict type. So I had to check if the `doc_type` property is 'CommCareUser' or not.